README.md

```diff
@@ -242,15 +179,17 @@ times before querying the next recommendations.
 
 ```python
 df = campaign.recommend(batch_quantity=3)
+print(df)
 ```
 
-For a particular random seed, the result could look as follows:
+> ```python

```

Comment:
not necessarily against this, but commenting for clarification:
we'd need to change this to 
```
~~~python
```
(or choose a different language here)

Reason: Our tests eck all code blocks in the docs for i) can be run correctly and ii) correct formatting (see pipeline which is currently failing). Blocks with "~~~" instead of "```" are interpreted as pseudocode and not checked for runnability

Reply:
it prob cant even be ~~~python, since anything with `python` will be checked for format and runnability (which will fail as this is not python code)
So prob something else like `commandline` should work

Reply:
fixed (I used `none` to suppress any syntax highlighting here, but omitting the lexer would also work in general)